src/cluster.c

```diff
@@ -1840,7 +1880,7 @@ void clearNodeFailureIfNeeded(clusterNode *node) {
      * node again. */
     if (nodeIsSlave(node) || node->numslots == 0) {
         serverLog(LL_NOTICE,
-            "Clear FAIL state for node %.40s: %s is reachable again.",
+            "Clear FAIL state for node %.40s:%s is reachable again.",
```

Comment:
The other use case for the human node name is that in these messages you could print out the human readable node name. I thought in a previous iteration of this PR we had done that.